### ISSUE TITLE
Audacity abruptly quits upon launch unless ran from the terminal

### ISSUE THREAD

**Issue by XA-Hydra** [pop-os/cosmic-epoch#506]
After installing Audacity on a clean install of today's Pop!_OS 24.04 Alpha, Audacity exits (crashes?) immediately after loading plugins for the first time. When re-launching Audacity, it displays the 'recover projects' dialog, but regardless of the option chosen, the application still quits.

Curiously, when running it directly from the terminal, the application seems to work fine. It then uses a light theme titlebar as opposed to the default dark titlebar (I am using the stock dark theme).

**Comment by mmstick** [comment]
Is this the flatpak or debian version?

**Comment by XA-Hydra** [comment]
debian

**Comment by XA-Hydra** [comment]
Just removed the .deb and installed via flatpak. Opening from the app store resulted in another abrupt exit after a second or so of splash screen. Running from the app drawer again, it DID work (at least as far as the main app window and an exit via the titlebar button). Running a second time didn't work, and now I am again at the 'recover projects' dialog and unable to proceed.

**Comment by authenticastrology** [comment]
Yes, I am noticing this too. It seems that a lot of the apps that are crashing on Cosmic are the ones that are launched from /bin
Timeshift
Zoom
Audacity
 All of the deb versions of these have their launchers in /bin. 

I am just a noob, but one with deductive reasoning...so feel free to tell me I am a noob. LOL

**Comment by authenticastrology** [comment]
Though off topic, actually Zoom and Timeshift seem to only open once and you then have a second to maximize them before they crash. Audacity seems to stay open.

**Comment by LiamillionSS** [comment]
Audacity issues have not been reported in a long time, I'm unable to replicate on arch or popos24.04
